### DISCUSSION TITLE
Cursor ripples option

### DISCUSSION THREAD

**Discussion by Puncake0** [ppy/osu#17861] [Ideas] [answered]
when i played osu! lazer i saw that this option was not in the game, a lot of players use it in stable
I would love to be able to play osu! lazer with cursor ripples :)
![unknown](https://user-images.githubusercontent.com/103921973/163766714-84517060-8f60-4ae0-8ca6-ff34cfb15ae3.png)
![unknown (1)](https://user-images.githubusercontent.com/103921973/163766879-abc11fab-71f0-4f28-a078-13ee82417c8c.png)

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/5070

**Comment by MatthieuHernandez** [comment]
Hello guys, hello @Puncake0 and @frenzibyte,

I also play osu!(stable) with the cursor ripples setting (named visualization of clicks in French) enabled.
UnfortunatelyIt appears that this setting is not available in osu!(lazer).

I tried to play without the cursor ripples but it really bothered me.
Consequently, I went back to play on osu!(stable) and it's a real shame.

Are there other people like me who would like to see this feature back in osu!(lazer)?

**Reply by Rekunan** [reply]
There's a PR implementing it that got merged #23342

**Comment by Kamoa** [comment]
I'm a big fan of cursor ripples and I've been hoping there could be an added option to match ripples color to current combo color. Basically to combine the aesthetic of hit lighting with the improved functionality of the ripples.
